platform/kube/controller.go

```diff
@@ -579,10 +579,10 @@ func (c *Controller) HostInstances(addrs map[string]bool) []*model.ServiceInstan
 }
 
 const (
-	// IstioServiceAccountPrefix is always the prefix for Istio service accounts.
-	IstioServiceAccountPrefix = "istio:"
-	// LocalDomain is the domain name Istio service account uses for internal traffic.
-	LocalDomain = "cluster.local"
+	// the URI scheme used to encode a Kubernetes service account
+	uriScheme = "spiffe"
+	// the domain name Istio service account uses for internal traffic
+	localDomain = "cluster.local"
```

Comment:
@kyessenov this encoded a service account, not a service. We are actually following how object path conversion used by the apiserver like https://kubernetes.io/docs/api-reference/v1.5/#create-482